QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
+
+        public void Start() {
+            while( _lineInput != "/exitQuestionbot" ) {
```

Comment:
We could do something like this...
```
public void Start(IConsole messageConsole) {
              while (ContinueReadLine()){
                        _lineInput = messageConsole.ReadLine();
                        NotifyAllListeners(_lineInput, messageConsole);
              }
         }

public bool ContinueReadLine(bool continue = true){
       return continue;
}
```

So it will always be true, but if we want to test it we can call ContinueReadLine and set it to false.
Are there any big disadvantages to calling ContinueReadLine() continuously?